webapp/src/pages/UserPage/UserPage.tsx

```diff
@@ -69,15 +135,16 @@ export default function UserPage() {
           YCoordinate: user?.avatarY ?? 0,
         }}
         button={
-          (blockRelation && {
-            disabled:
-              (blockRelation.amIBlocked || blockRelation.isUserBlocked) ??
-              false,
-            variant: ButtonVariant.SUBMIT,
-            iconVariant: IconVariant.SEND,
-            children: 'Send message',
-            onClick: () => navigate(`${CHAT_URL}/${user?.username}`),
-          }) ||
+          (!overridePermissions &&
```

Comment:
Maybe too many things diverge in this page from the regular user page to the admin version. Maybe it would be worth to make two different Elements.

Reply:
I think it's ok as it is now. Maybe if we add more things, it's better to create different pages. Up to you :+1: